web/assets/css/bundle_card.css

```diff
@@ -0,0 +1,151 @@
+.bundle-card-line {
+    overflow: hidden;
+    margin-top: 20px;
+}
+
+.bundle-card-line-data {
+    background: rgb(241, 241, 241);
+    overflow: hidden;
+    border-top-left-radius: 5px;
+    border-top-right-radius: 5px;
+    padding: 20px 5px;
+}
+
+.bundle-card-line-content h4 {
```

Comment:
Change to h2 (ref: https://github.com/ezsystems/ezplatform-com/pull/80/files#diff-205461f0c399ee7cdb4ed7a7eda49f76R4)